third-party/zen-pmu-workaround/zen_workaround.c

```diff
@@ -0,0 +1,114 @@
+#include <linux/module.h>
```

Comment:
Looks like kernel code now uses this as a header:

// SPDX-License-Identifier: GPL-2.0-only

I'd be fine with using whatever license makes this easier, but the module does need the 
`MODULE_LICENSE("GPL");` for the pm_notifier and probe registrations.

Reply:
Are you suggesting a change here?

Reply:
I was just suggesting to put `// SPDX-License-Identifier: GPL-2.0-only` as a header in this file.